Gleb_Yelizov/main.js

```diff
@@ -27,3 +27,75 @@ function render() {
     main.append(ul);
     document.body.append(main);
 }
+//-------------------------------------------------------------------lesson_04 Homework------------------------------------------------
+
+//----------------get Paired array-----------------
+
+function getPaired(arr) {
+    let paired = [];
+    
+    for (let i = 0; i <= arr.length; i++) {
+        if ((arr[i] % 2 == 0) && !(arr[i] == 0)) {
```

Comment:
no need to move the `!` out from comparison expression, you could left if like that
```suggestion
        if ((arr[i] % 2 === 0) && arr[i] !== 0) {
```
also, please notice that I've changed `==` to `===` in the first expression